python/invoke_release/tasks.py

```diff
@@ -1198,16 +1259,28 @@ def release(_, verbose=False, no_stash=False):
 
         _pre_commit(__version__, release_version)
 
+        if USE_PULL_REQUEST:
+            current_branch_name = _get_branch_name(verbose)
+            branch_name = 'invoke-release-{}-{}'.format(current_branch_name, release_version)
+            _create_branch(verbose, branch_name)
         _commit_release_changes(release_version, cl_message, verbose)
 
         _pre_push(__version__, release_version)
 
-        _tag_branch(release_version, cl_message, verbose)
+        if USE_TAG:
+            _tag_branch(release_version, cl_message, verbose)
         pushed_or_rolled_back = _push_release_changes(release_version, branch_name, verbose)
 
+        if USE_PULL_REQUEST:
+            _checkout_branch(verbose, BRANCH_MASTER)
+
         _post_release(__version__, release_version, pushed_or_rolled_back)
 
-        _standard_output('Release process is complete.')
+        if USE_PULL_REQUEST:
+            _standard_output('You\'re almost done! The release process will be complete when you create '
```

Comment:
Normally, we say "always use single quotes." However, our exception to that is that you can use double quotes to avoid having to escape single quotes within the string. So it's fine to make this double quotes (but I won't block over it).